spec/factory_specs/fois_spec.rb

```diff
@@ -127,7 +127,7 @@
         expect(managing_assignment.state).to eq 'accepted'
         expect(managing_assignment.team).to eq team_disclosure_bmt
 
-        expect(kase.transitions.size).to eq 0
+        expect(kase.transitions.size).to eq 1
```

Comment:
Did Alex know that this change would affect ALL case types, and not just Offender SARs? I think it's great, but just checking... :)

Reply:
@Polsonby  yes I did mention this to her, she said it is good to have "created event" in the case history, as it is one of important feedback from the testing running with Branton officers,  so my idea would be to deploy this ticket to dev and ask her to give a run for all those cases (to be honest, there is no any change to other case types only change is this new event)